app/Services/Plans/PlanService.php

```diff
@@ -24,14 +24,12 @@ public function __construct()
      * @param int $plan_id
      * @param Bible $bible
      * @param int $user_id
-     * @param int $show_details - If it is true, it will create the verse_text property for each play list item that belong to a day
```

Comment:
I am confused. show_details was removed from PlanService, but it is still included in the call to PlanService, higher in this PR